public/app/features/explore/Wrapper.tsx

```diff
@@ -86,22 +118,56 @@ class WrapperUnconnected extends PureComponent<Props> {
     document.title = documentTitle;
   }
 
+  updateSplitSize(rightPaneWidth: number) {
+    const evenSplitWidth = window.innerWidth / 2;
```

Comment:
Shouldn't we use container's width, not the window width?

Reply:
This boolean is to know what the threshold is for an even split. The `rightPaneWidth` parameter is the container's width.

Reply:
I meant it just happens that Wrapper spans the window width, but if we ever change it the threshold calculations will be incorrect. Never mind, I think it was like this before so you can leave it.